Nguyen_Ken/Assignments/Flask with MySQL/the_wall/templates/wall.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang='en'>
+<head>
+	<title>KenBook Wall</title>
+	<meta charset='UTF-8'>
+	<meta name='description' content='KenBook Wall'>
+	<link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/styles.css')}}">
+</head>
+<body>
+	<div class='wallWrapper'>
+		{%if session['user']%}
+			{%for user in session['user']%}
+				<h1>Welcome {{user['first_name']}}</h1>
+				<a href='/logout' id='logout'>Log Out</a>
+			{%endfor%}
+		{%endif%}
+<!-- 			{%if user%}
```

Comment:
Just a note, even though you commented this out w/ html comments, the python code will still be executed and printed _into_ the html source